src/client.rs

```diff
@@ -929,3 +927,13 @@ impl Client {
         }
     }
 }
+
+/// Response returned by [`Client::new_window()`] method.
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct NewWindowResponse {
```

Comment:
This feels like it should go in `wd`. It also feels like it should have `#[non_exhaustive]`.

Reply:
> This feels like it should go in `wd`.

I've put it here, because it's not the type/structure explicitly declared by [the spec](https://www.w3.org/TR/webdriver), but a helper type for representing result of the concrete `Client` method. I'm used to keep such things near to the method requiring them.

Though, I'm OK with moving it to `wd` if you prefer so.

>  It also feels like it should have `#[non_exhaustive]`.

It's completely defined by [spec](https://w3c.github.io/webdriver/#dfn-new-window):
> 8. Let result be a new JSON Object initialised with:
> 
> "handle"
>     The value of handle. 
> "type"
>     The value of type. 

So it won't change unless the spec changes. And migrating to new spec is breaking change anyway.

Considering this, I think `#[non_exhaustive]` here will only create unconveniences/uncertainty for user code. Unless we want to propose here some custom extended interface, not repeat the WebDriver spec exactly.

Reply:
I buy both of those arguments :+1: